privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -764,15 +767,25 @@ def loadConfig(self, config):
         self.resolverId = self.uri
         self.authtype = config.get("AUTHTYPE", AUTHTYPE.SIMPLE)
         self.tls_verify = is_true(config.get("TLS_VERIFY", False))
-        # Fallback to TLSv1. (int: 3, TLSv1.1: 4, v1.2: 5)
-        self.tls_version = int(config.get("TLS_VERSION") or ssl.PROTOCOL_TLSv1)
-
+        # Fallback to TLSv1. (int: 3, TLSv1.1: 4, v1.2: 5, TLS negotiation: 2)
+        self.tls_version = int(config.get("TLS_VERSION") or DEFAULT_TLS_PROTOCOL)
```

Comment:
I think the config should now (with the migration script) always contain a TLS_VERSION (at least 1.0 for the old configs).
I am a bit puzzled, if we would have to reset this here to ssl.PROTOCOL_TLSv1.